include/swift/AST/IRGenOptions.h

```diff
@@ -217,6 +223,8 @@ class IRGenOptions {
   /// Whether we should embed the bitcode file.
   IRGenEmbedMode EmbedMode : 2;
 
+  IRGenLLVMLTOKind LLVMLTOKind: 2;
```

Comment:
This code has not been run through git-clang-format. Please do so before this lands!